api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -577,7 +579,7 @@ def __init__(
             experimental
         """
         self.exp_uri = experiment.uri
-        self.aws_region = experiment.context.tiledb_ctx.config().get("vfs.s3.region")
+        self.aws_region = experiment.context.tiledb_ctx#.config().get("vfs.s3.region")
```

Comment:
This seems a bit suspicious - can you remind me what exactly required this change?